collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -171,16 +171,7 @@ public void setExternalDataManager(ExternalDataManager externalDataManager) {
     }
 
     public String getVersionedAppName() {
-        String versionName = "";
-        try {
-            versionName = getPackageManager()
-                    .getPackageInfo(getPackageName(), 0)
-                    .versionName;
-            versionName = " " + versionName.replaceFirst("-", "\n");
```

Comment:
I think you need to test this without having git in your path. I think if git is not in your path, the `getVersionName` in the build.gradle will return null (it will go to the catch section). And then null.replaceFirst() will throw NPE in this code.